app/src/test/java/org/example/screens/InfoAlert.java

```diff
@@ -0,0 +1,30 @@
+package org.example.screens;
+
+import com.codeborne.selenide.appium.SelenideAppiumElement;
+import io.appium.java_client.pagefactory.AndroidFindBy;
+
+import static com.codeborne.selenide.Condition.disappear;
+import static com.codeborne.selenide.Condition.visible;
+import static com.codeborne.selenide.appium.AppiumClickOptions.tap;
+import static com.codeborne.selenide.appium.SelenideAppium.$;
+
+public class InfoAlert {
+    @AndroidFindBy(id = "android:id/message")
+    private SelenideAppiumElement msgText;
+
+    @AndroidFindBy(id = "android:id/button1")
+    private SelenideAppiumElement okBtn;
+
+    public String getMsgText() {
+        return  msgText.shouldBe(visible).getText();
+    }
+
+    public void clickOkBtn() {
+        okBtn.shouldBe(visible).click(tap());
+    }
+
+    public void checkDisappear() {
+        okBtn.shouldBe(disappear);
+        msgText.shouldBe(disappear);
```

Comment:
```suggestion
        okBtn.should(disappear);
        msgText.should(disappear);
```
Предлагаю другой вариант метода, чтобы складывалась правильная фраза

Reply:
> Предлагаю другой вариант метода, чтобы складывалась правильная фраза

там где disappear использую should -- сделано